### PR TITLE
Ffm 1277

### PR DESCRIPTION
**Pull request by milos85vasic-harness**
This PR contains changes required for the Docker container to use SDK params via env. variables on runtime.

### CODE REVIEW

File: test_wrapper/Dockerfile [hunk 1]
```diff
@@ -28,14 +26,14 @@ RUN mkdir CFAndroidSDK && cd ./CFAndroidSDK && \
...
     echo "sdk.dir=/Android" > local.properties && \
-    echo "{\"selfTest\": $SELF_TEST, \"port\": $PORT, \"apiKey\": \"$API_KEY\", \"logger\": \"filesystem\"}" \
-        > "test_wrapper/wrapper.json" && \
-        echo "wrapper.json:" && cat "test_wrapper/wrapper.json" && \
     ../gradle-7.0/bin/gradle wrapper --gradle-version 7.0 && ../gradle-7.0/bin/gradle --stop
 
 EXPOSE $PORT
 
 CMD cd ./CFAndroidSDK && git fetch && git pull && \
+    echo "{\"selfTest\": $SELF_TEST, \"port\": $PORT, \"sdkKey\": \"$SDK_KEY\", \"logger\": \"filesystem\", \"enableStreaming\": $ENABLE_STREAMING, \"eventUrl\": \"$EVENT_URL\", \"sdkBaseUrl\": \"$SDK_BASE_URL\"}" \
```
Comment (Andrew-Hayes) [inline test_wrapper/Dockerfile:34]: will the sdkKey part of this just create an empty entry?

Comment (milos85vasic-harness) [reply test_wrapper/Dockerfile:34]: It should not. It will populate with the environment variable value of the SDK_KEY.

Comment (Andrew-Hayes) [reply test_wrapper/Dockerfile:34]: yes, you are right! my bad

**Review by Andrew-Hayes — APPROVED** [review]
LGTM

**Pull request merged**